buf-ledger-api.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: v1beta1
+
+build:
+  roots:
+    - ledger-api/grpc-definitions
+    - ledger-api/3rdparty/grpc-definitions
+
+breaking:
+  use:
+    - WIRE_JSON
+    - FIELD_NO_DELETE
```

```suggestion
    # Set to FILE to ensure that all breakages of the gRPC Ledger API are explicitly
    # discussed with Engineering and Product Leadership.
    - FILE
```

If `FILE` works then, I'd really like to use it; even though we don't guarantee it explicitly to our customers.

My rationale is that backwards-compatibility for the Ledger API matters for the UX of our customers, and is something we want to make a very explicit choice to break. Requiring `FILE` here ensures that the required discussions will happen.

---

99df542